src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add: function(num) {
```

Comment:
Fix it everywhere
<img width="1122" alt="image" src="https://github.com/mate-academy/js_advanced_calculator/assets/64750958/d5236bc7-cf31-4eea-b16c-f6655e9db0d9">